### PR TITLE
Run cvuqdisk search as root

### PR DESCRIPTION
**Pull request by mfielding**
In some cases (possibly involving a re-run of the install without an intervening `cleanup-oracle.sh` run), the grid home gets permissions locked down, so that the ansible user can't read files here.  This change adds a become: (ie sudo) to run the find command as root.  The actual installation already runs as root.

Internal bug reference: b/348132168

### CODE REVIEW

**Comment by alex-basinov** [comment]
The test run failed with the following error:

unzip:  cannot find or open /u01/oracle_install/V982068-01.zip, /u01/oracle_install/V982068-01.zip.zip or /u01/oracle_install/V982068-01.zip.ZIP.\n/bin/bash: line 3: gsutil: command not found

Is the failure related to the change?

**Comment by mfielding** [comment]
Yeah, digging into the error;  the immediate cause is missing gsutil, and looks like this happened because all the CentOS stream 8 repos were unceremoniously deleted a few weeks ago: https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/  Working on a hack that will avoid depending on upstream packages, to unblock this: PR#168

**Review by alex-basinov — DISMISSED** [review]

File: presubmit_tests/test-pr-poc.sh
Comment (patelmithlesh) [inline presubmit_tests/test-pr-poc.sh:1]: /lgtm

Comment (patelmithlesh) [reply presubmit_tests/test-pr-poc.sh:1]: /lgtm

**Review by patelmithlesh — COMMENTED** [review]
/lgtm

**Review by patelmithlesh — APPROVED** [review]
LGTM

**Review by patelmithlesh — APPROVED** [review]
LGTM

**Review by patelmithlesh — DISMISSED** [review]
lgtm

**Comment by patelmithlesh** [comment]
lgtm

**Review by patelmithlesh — DISMISSED** [review]
lgtm

**Comment by patelmithlesh** [comment]
lgtm

**Review by patelmithlesh — APPROVED** [review]
lgtm

**Comment by mfielding** [comment]
Why all the lgtms and dismissals here?  Looks like a KI with GitHub (https://github.com/orgs/community/discussions/58535) with a workaround of closing PR, reopening, and getting approval again.

**Pull request merged**